packages/yaml/test/test.js

```diff
@@ -59,8 +59,11 @@ test('resolves extensionless imports in conjunction with nodeResolve plugin', as
 });
 
 test('applies the optional transform method to parsed YAML', async (t) => {
-  const transform = (data) => {
+  const transform = (data, filePath) => {
+    // check that transformer is passed a correct file path
+    t.truthy(typeof filePath === 'string' && filePath.endsWith('.yaml'));
```

Comment:
Let's just use `t.true` rather than `truthy`. Passing the file path as a second argument will print it if the assertion fails.
```suggestion
    t.true(typeof filePath === 'string' && filePath.endsWith('.yaml'), filePath);
```
Once this change is made to both assertions we're good to merge

Reply:
done